CHANGELOG.md

```diff
@@ -12,6 +12,12 @@
 ### Changed
 - tauri: replace `tauri-plugin-shell` with `tauri-plugin-opener` #4699
 - new button styles #4741
+- Update `@deltachat/stdio-rpc-server` and `deltachat/jsonrpc-client` to `1.157.1`
+  - Ignore encryption preferences
```

Comment:
this point might need clarification

Reply:
See https://github.com/chatmail/core/pull/6612

Reply:
@link2xt  Is that a "breaking change" in some scenarios?

Reply:
> We should talk to them about our plans for PGP-contacts and multitransport anyway. For most users it does not matter.

Does this means, we should wait with the core update and the next release with core 1.157.x ?

Reply:
If it's only on Android so desktop is not a use case for them?

Reply:
@nicodh please clarify it in a way that people can not think that this means less encryption.

Reply:
maybe just remove this misleading line when merging #4785, the description there should be enough